tutorials/hyperformula-integration.html

```diff
@@ -0,0 +1,259 @@
+
+    <h3 id="overview">Overview</h3>
+    <p>
+      The <i>Formulas</i> plugin provides you an extensive calculation capabilities based on formulas with the spreadsheet notation. Under the hood it uses an engine called <a href="https://handsontable.github.io/hyperformula/">HyperFormula</a> created by the Handsontable team as an independent library to help developers build complex data management apps.
+    </p>
+    <p>
+      This plugin comes with a library of 386 functions grouped into categories, such as Math and trigonometry, Engineering, Statistical, Financial, and Logical. Thanks to them you can create complex data entry rules in business apps, and many more. Below are some ideas on what you can do with it:
+    </p>
+    <p>
+      <ul>
+        <li>Fully-featured spreadsheet apps</li>
+        <li>Smart documents</li>
+        <li>Educational apps</li>
+        <li>Business logic builders</li>
+        <li>Forms and form builders</li>
+        <li>Online calculators</li>
+        <li>Low connectivity apps</li>
+      </ul>
+    </p>
+
+    <h3 id="features">Features</h3>
+    <p>
+      <ul>
+        <li>High-speed formula calculations</li>
+        <li>Function syntax compatible with Excel and Google Sheets</li>
+        <li>A library of built-in functions available in 16 languages</li>
+        <li>Support for wildcard characters</li>
+        <li>Support for CRUD operations</li>
+        <li>Support for cross-sheet references</li>
+        <li>Support for multiple Handsontable instances</li>
+        <li>Uses GPU acceleration for better performance</li>
+      </ul>
+    </p>
+
+    <h3 id="available-options-and-methods">Available options and methods</h3>
+    <p>
+      For the list of available settings and methods, visit the <a href="/docs/next/Formulas.html">API reference</a>.
+    </p>
+
+    <h3 id="available-functions">Available functions</h3>
+    <p>
+      This plugin inherits the calculation powers from <i>HyperFormula</i>, so the complete functions reference can be found in the <a href="https://handsontable.github.io/hyperformula/guide/built-in-functions.html">HyperFormula documentation</a>.
+    </p>
+
+    <h3 id="basic-multi-sheet-example">Basic multi-sheet example</h3>
+
+    <p>It is possible to use the plugin in single sheet mode or with multiple Handsontable instances with cross-sheet references.</p>
+    <p>Double click on a cell to open the editor and preview the formula.</p>
+
+    <div id="example-basic-multi-sheet-1" class="hot"></div>
+    <br/>
+    <div id="example-basic-multi-sheet-2" class="hot"></div>
+
+    <div class="codeLayout">
+      <script>
+        var data1 = [
+          ['10.26', '', 'Sum', '=SUM(A:A)'],
+          ['20.12', '', 'Average', '=AVERAGE(A:A)'],
+          ['30.01', '', 'Median', '=MEDIAN(A:A)'],
+          ['40.29', '', 'MAX', '=MAX(A:A)'],
+          ['50.18', '', 'MIN', '=MIN(A:A)'],
+        ];
```

Comment:
`COUNTBLANK` does not count an empty strings so for better number representation I suggest to change the date of the second column.
```suggestion
        var data1 = [
          ['10.26', null, 'Sum', '=SUM(A:A)'],
          ['20.12', null, 'Average', '=AVERAGE(A:A)'],
          ['30.01', null, 'Median', '=MEDIAN(A:A)'],
          ['40.29', null, 'MAX', '=MAX(A:A)'],
          ['50.18', null, 'MIN', '=MIN(A:A)'],
        ];
```